### ISSUE TITLE
Documenting a Test Run

### ISSUE THREAD

**Issue by codenithin** [ehmatthes/django-simple-deploy#255]
Using macOS Ventura 13.4.1
Own Project - https://github.com/codenithin/Simple-Invoice-Crud-app
Dependency Management - Using pip and Requirements.txt
Platform - Fly.io

Liked the Web-CLI concept of Fly.io but could not get my app working on Web CLI directly nor through django-simple-deploy

Issue Faced with django-simple-deploy
CommandError: 
--- A Fly.io region was not found. ---

We need to know what region the app is going to be deployed to.
We could not find a region in the output of:

$ flyctl regions list -a snowy-glade-4907

**Comment by ehmatthes** [comment]
Thanks for this! I think I have seen this issue before. I'll look into it and get back to you.

**Comment by ehmatthes** [comment]
I believe this issue arises because Fly has deprecated most of the `fly regions` [command](https://fly.io/docs/flyctl/regions/). This command was used to identify the region that the app was created in, so we can create a db in the same region.

I believe the dev version of this project (current HEAD of main branch) does not face this issue. I made a [quick fix](https://github.com/ehmatthes/django-simple-deploy/blob/main/simple_deploy/management/commands/fly_io/deploy.py#L503) in June to return the region `sea` if `fly regions` returns nothing. That works for me, but it's not ideal for people with regions much closer to them. I assume it's also much better to have your db in the same region as your app, so this is definitely a temp fix. #256 focuses on the proper fix to this issue.

If you are curious to see a successful deployment, I would guess installing simple_deploy from this repo instead of PyPI would bypass this issue:

```sh
$ pip install 'django-simple-deploy @ https://github.com/ehmatthes/django-simple-deploy
```

Also it's always worth repeating, if you haven't done so already make sure you deleted the app and any other resources that were created for this deployment.

**Comment by codenithin** [comment]
Thanks Eric. I will try this.

**Comment by ehmatthes** [comment]
This should be fixed by the 0.5.16 release, which was just pushed. If you try another push and it fails, please feel free to reopen this issue, or open a new issue. Thanks again!
